recipes/bioconductor-interest/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set version = '1.2.2' %}
+{% set name = "IntEREst" %}
+{% set bioc = "3.6" %}
+
+package:
+  name: 'bioconductor-{{ name|lower }}'
+  version: {{ version }}
+
+source:
+  fn: '{{ name }}_{{ version }}.tar.gz'
+  url:
+    - 'http://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
+  sha256: b8560d301118db28c409fb422a7457bc16f052a2e8a97a12cfc6f122569d62b2
+
+build:
+  number: 1
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base >=3.4
+    - r-seqinr
+    - r-dbi
+    - r-rmysql
+    - bioconductor-dexseq
+    - bioconductor-biocparallel
+    - bioconductor-genomeinfodb
+    - bioconductor-biocgenerics
+    - bioconductor-genomicranges
+    - bioconductor-rsamtools
+    - bioconductor-summarizedexperiment
+    - bioconductor-edger
+    - bioconductor-s4vectors
+    - bioconductor-deseq2
+    - bioconductor-genomicalignments
+    - bioconductor-iranges
+    - bioconductor-genomicfeatures
+    - bioconductor-biostrings
+    - bioconductor-seqlogo
+
+  run:
+    - r-base >=3.4
+    - r-seqinr
+    - r-dbi
+    - r-rmysql
+    - bioconductor-dexseq
+    - bioconductor-biocparallel
+    - bioconductor-genomeinfodb
+    - bioconductor-biocgenerics
+    - bioconductor-genomicranges
+    - bioconductor-rsamtools
+    - bioconductor-summarizedexperiment
+    - bioconductor-edger
+    - bioconductor-s4vectors
+    - bioconductor-deseq2
+    - bioconductor-genomicalignments
+    - bioconductor-iranges
+    - bioconductor-genomicfeatures
+    - bioconductor-biostrings
+    - bioconductor-seqlogo
+
+test:
+  commands:
+    - '$R -e "library(''{{ name }}'')"'
+
```

Comment:
can you add a DOI please into the extra section if possible?

Reply:
let me know if this is ok